app/containers/App/reducer.js

```diff
@@ -51,7 +52,15 @@ const createSubmit = ({ date, ...rest }) => ({
 function contestsPageReducer(state = initialState, action) {
   switch (action.type) {
     case LOGIN_SUCCESS:
-      return state.set('user', fromJS(action.user));
+      return state.mergeDeep({
+        user: fromJS(action.user),
+        logged: true,
```

Comment:
nie jestem pewien czy osobna flaga logged to jest dobry pomysł ponieważ to jest kolejne pole które trzeba utrzymywać. Wydaje mi się że lepszym pomysłem jest decydowanie w selektorze czy user jest zalogowany na podstawie obiektu `user`. I nie wiem czy nie istnieje gdzieś już jakaś funkcja do sprawdzania czy user nie jest zalogowany.